sdk/src/system_transaction.rs

```diff
@@ -15,10 +15,58 @@ pub fn create_account(
     lamports: u64,
     space: u64,
     program_id: &Pubkey,
+) -> Transaction {
+    generate_create_account_tx(
+        from_keypair,
+        to,
+        recent_blockhash,
+        lamports,
+        space,
+        program_id,
+        false,
+    )
+}
+
+pub fn create_rent_exempted_account(
+    from_keypair: &Keypair,
+    to: &Pubkey,
+    recent_blockhash: Hash,
+    lamports: u64,
+    space: u64,
+    program_id: &Pubkey,
+) -> Transaction {
+    generate_create_account_tx(
+        from_keypair,
+        to,
+        recent_blockhash,
+        lamports,
+        space,
+        program_id,
+        true,
+    )
+}
+
+fn generate_create_account_tx(
+    from_keypair: &Keypair,
+    to: &Pubkey,
+    recent_blockhash: Hash,
+    lamports: u64,
+    space: u64,
+    program_id: &Pubkey,
+    should_rent_exempt: bool,
 ) -> Transaction {
     let from_pubkey = from_keypair.pubkey();
-    let create_instruction =
-        system_instruction::create_account(&from_pubkey, to, lamports, space, program_id);
+    let create_instruction = if should_rent_exempt {
```

Comment:
at this point, I'd make the system_instruction::generate_XXX function pub(crate)...